oracle/oracle.go

```diff
@@ -441,3 +453,152 @@ func (o *oracleOps) Delete(volumeID string) error {
 	}
 	return nil
 }
+
+// Attach volumeID, accepts attachOptions as opaque data
+// Return attach path.
+func (o *oracleOps) Attach(volumeID string, options map[string]string) (string, error) {
+	o.mutex.Lock()
+	defer o.mutex.Unlock()
+
+	devices, err := o.FreeDevices([]interface{}{}, "")
+	if err != nil {
+		return "", err
+	}
+
+	for _, device := range devices {
```

Comment:
Since, we want to use [Consistent Device Paths](https://docs.oracle.com/en-us/iaas/Content/Block/References/consistentdevicepaths.htm), we need to mention device name.

Reply:
Got it